aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -373,4 +384,12 @@ public String dynamoDbTableName() {
   public void setDynamoDbTableName(String name) {
     this.dynamoDbTableName = name;
   }
+
+  public boolean isS3ChecksumEnabled() {
+    return this.isS3ChecksumEnabled;
+  }
+
+  public void setS3ChecksumEnabled(boolean eTagCheckEnabled) {
```

Comment:
nit: `eTagCheckEnabled` was not updated from the old name, maybe just `enabled`?